matching-core/src/main/java/com/graphhopper/matching/mapmatchingutils/Distributions.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package com.graphhopper.matching;
+package com.graphhopper.matching.mapmatchingutils;
```

Comment:
I know this belongs to the previous PR but just wanted to mention it here: I would prefer just "com.graphhopper.matching.util" which would make it more align with e.g. "com.graphhopper.util" or "com.graphhopper.routing.util"